test/mocha.opts

```diff
@@ -1,3 +1,3 @@
 test/specs/**/*.js
 -R dot
--u bdd
+-u bdd
```

Comment:
I edited this while developing and accidentally pushed the change, then restored original content.